backend/sqlx-data.json

```diff
@@ -162,6 +162,16 @@
       "nullable": []
     }
   },
+  "56b986f2a2b7091d9c3acdd78f75d9842242de1f4da8f3672f2793d9fb256928": {
```

Comment:
Confirmation: This is the generation of sqlx right.

Reply:
yes